src/core/ext/filters/client_channel/lb_policy/ring_hash/ring_hash.cc

```diff
@@ -314,9 +314,9 @@ RingHash::Ring::Ring(RingHash* parent,
     const ServerAddressWeightAttribute* weight_attribute = static_cast<
         const ServerAddressWeightAttribute*>(sd->address().GetAttribute(
         ServerAddressWeightAttribute::kServerAddressWeightAttributeKey));
+    auto addr_str = grpc_sockaddr_to_string(&sd->address().address(), false);
     AddressWeight address_weight;
-    address_weight.address =
-        grpc_sockaddr_to_string(&sd->address().address(), false);
+    address_weight.address = std::string(addr_str.value());
```

Comment:
No need for the explicit `std::string()` here, since that's already the type of `addr_str.value()`.

Reply:
Done.